client/components/calypso-i18n-provider/index.tsx

```diff
@@ -25,7 +27,11 @@ const CalypsoI18nProvider: React.FunctionComponent = ( { children } ) => {
 		};
 	}, [] );
 
-	return <I18nProvider localeData={ localeData }>{ children }</I18nProvider>;
+	return (
+		<LocaleProvider localeSlug={ localeSlug || i18nDefaultLocaleSlug }>
```

Comment:
Because `<CalypsoI18nProvider>` is very app specific I'd use the `i18n_default_locale_slug` config instead of the default that comes from the package.

Pretty nit picky given it's the same value and `i18n_default_locale_slug` will likely never change.

```suggestion
		<LocaleProvider localeSlug={ localeSlug || config( 'i18n_default_locale_slug' ) }>
```

Reply:
Happy to use this for now but realistically the config values need to be removed now that they are required outside of calypso package. Why I moved all of them to hard coded values in i18n-utils package.